airbyte-integrations/connectors/destination-gcs/src/main/java/io/airbyte/integrations/destination/gcs/GcsDestination.java

```diff
@@ -52,10 +56,21 @@ public AirbyteConnectionStatus check(final JsonNode config) {
       S3Destination.testMultipartUpload(s3Client, destinationConfig.getBucketName(), destinationConfig.getBucketPath());
 
       return new AirbyteConnectionStatus().withStatus(Status.SUCCEEDED);
+    } catch (final AmazonS3Exception e) {
+      LOGGER.error("Exception attempting to access the Gcs bucket: {}", e.getMessage());
+      LOGGER.error("Please make sure you account has all of these roles: " + EXPECTED_ROLES);
```

Comment:
I don't think this log is needed here. Lack of permissions is a special case as one of the possible reasons for AmazonS3Exception being thrown, but we need to provide the exact reason

Reply:
removed it